src/main/g8/project/ProjectPlugin.scala

```diff
@@ -0,0 +1,70 @@
+import sbt._
+import sbt.Keys._
+import org.scalafmt.sbt.ScalafmtPlugin.autoImport._
+import sbt.{AutoPlugin, PluginTrigger}
+
+object ProjectPlugin extends AutoPlugin {
+
+  override def trigger: PluginTrigger = allRequirements
+
+  object autoImport {
+
+    lazy val V = new {
+      val fs2            = "0.10.1"
+      val log4cats       = "0.1.0"
+      val logbackClassic = "1.2.3"
+      val circe          = "0.10.0-M1"
+      val freestyleRPC   = "0.14.0"
+      val http4s         = "0.18.15"
```

Comment:
Are we using http4s?

Reply:
Nope. There are come version that can be dropped off here, thanks.